code/core/write_sdl_ntuple.cc

```diff
@@ -201,35 +203,32 @@ void setOutputBranches(SDL::Event* event)
         if (trk.sim_event()[isimtrk] != 0)
             continue;
 
-        ana.tx->pushbackToBranch<float>("sim_pt", trk.sim_pt()[isimtrk]);
-        ana.tx->pushbackToBranch<float>("sim_eta", trk.sim_eta()[isimtrk]);
-        ana.tx->pushbackToBranch<float>("sim_phi", trk.sim_phi()[isimtrk]);
-        ana.tx->pushbackToBranch<float>("sim_pca_dxy", trk.sim_pca_dxy()[isimtrk]);
-        ana.tx->pushbackToBranch<float>("sim_pca_dz", trk.sim_pca_dz()[isimtrk]);
-        ana.tx->pushbackToBranch<int>("sim_q", trk.sim_q()[isimtrk]);
-        ana.tx->pushbackToBranch<int>("sim_event", trk.sim_event()[isimtrk]);
-        ana.tx->pushbackToBranch<int>("sim_pdgId", trk.sim_pdgId()[isimtrk]);
+        ana.tx->pushbackToBranch<float>("SimTrack_pt", trk.sim_pt()[isimtrk]);
+        ana.tx->pushbackToBranch<float>("SimTrack_eta", trk.sim_eta()[isimtrk]);
+        ana.tx->pushbackToBranch<float>("SimTrack_phi", trk.sim_phi()[isimtrk]);
+        ana.tx->pushbackToBranch<float>("SimTrack_dxy", trk.sim_pca_dxy()[isimtrk]);
+        ana.tx->pushbackToBranch<float>("SimTrack_dz", trk.sim_pca_dz()[isimtrk]);
+        ana.tx->pushbackToBranch<int>("SimTrack_charge", trk.sim_q()[isimtrk]);
+        ana.tx->pushbackToBranch<int>("SimTrack_ppVtx", trk.sim_event()[isimtrk]);
+        ana.tx->pushbackToBranch<int>("SimTrack_pdgID", trk.sim_pdgId()[isimtrk]);
 
         // For vertex we need to look it up from simvtx info
         int vtxidx = trk.sim_parentVtxIdx()[isimtrk];
-        ana.tx->pushbackToBranch<float>("sim_vx", trk.simvtx_x()[vtxidx]);
-        ana.tx->pushbackToBranch<float>("sim_vy", trk.simvtx_y()[vtxidx]);
-        ana.tx->pushbackToBranch<float>("sim_vz", trk.simvtx_z()[vtxidx]);
-
-        // The trkNtupIdx is the idx in the trackingNtuple
-        ana.tx->pushbackToBranch<float>("sim_trkNtupIdx", isimtrk);
+        ana.tx->pushbackToBranch<float>("SimTrack_vx", trk.simvtx_x()[vtxidx]);
+        ana.tx->pushbackToBranch<float>("SimTrack_vy", trk.simvtx_y()[vtxidx]);
+        ana.tx->pushbackToBranch<float>("SimTrack_vz", trk.simvtx_z()[vtxidx]);
 
         // Increase the counter for accepted simtrk
         n_accepted_simtrk++;
     }
 
     // Intermediate variables to keep track of matched track candidates for a given sim track
-    std::vector<int> sim_TC_matched(n_accepted_simtrk);
-    std::vector<int> sim_TC_matched_mask(n_accepted_simtrk);
-    std::vector<int> sim_TC_matched_for_duplicate(trk.sim_pt().size());
+    std::vector<int> SimTrack_TC_match_for_duplicate(trk.sim_pt().size());
+    std::vector<std::vector<int>> SimTrack_TC_idx(n_accepted_simtrk);
```

Comment:
Jonathan and I are designing branch name convention.
So that from now on every branch can be consistent.
https://docs.google.com/spreadsheets/d/1k42XvQys5FqR8mRiQcg3NyVo0Cn3Nw8wAWHd2XyjB7M/edit#gid=0
It's part of that on-going work.... I figured I change this portion first while at it.